### PR TITLE
Add tests for software.amazon.kinesis.leases.DynamoUtils

### PR DESCRIPTION
**Pull request by sudo-jaa**
[insert PR text]

### CODE REVIEW

File: amazon-kinesis-client/src/test/java/software/amazon/kinesis/leases/DynamoUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    final HashMap<String, AttributeValue> dynamoRecord3 = new HashMap<String, AttributeValue>();
+    Assert.assertNull(DynamoUtils.safeGetString(dynamoRecord3, ""));
+
+    final HashMap<String, AttributeValue> dynamoRecord4 = new HashMap<String, AttributeValue>();
+    AttributeValue attribute = DynamoUtils.createAttributeValue("test");
+    dynamoRecord4.put("#", attribute);
+    Assert.assertEquals("test", DynamoUtils.safeGetString(dynamoRecord4, "#"));
+  }
+
```
Comment (paulkennethkent) [inline amazon-kinesis-client/src/test/java/software/amazon/kinesis/leases/DynamoUtilsTest.java:31]: minor spacing issue

**Review by paulkennethkent — APPROVED** [review]

File: amazon-kinesis-client/src/test/java/software/amazon/kinesis/leases/DynamoUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
+package software.amazon.kinesis.leases;
+
+import org.junit.Assert;
+import org.junit.Test;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.ArrayList;
```
Comment (petersear) [inline amazon-kinesis-client/src/test/java/software/amazon/kinesis/leases/DynamoUtilsTest.java]: I think imports of the same type should be arranged alphabetically ie. ArrayList before Collection and HashMap.

**Review by petersear — APPROVED** [review]
I noticed some tests have been dropped since the first PR and presume that is deliberate.
Otherwise, I just noticed a possible required reordering of import statements.

File: amazon-kinesis-client/src/test/java/software/amazon/kinesis/leases/DynamoUtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+    final HashMap<String, AttributeValue> dynamoRecord2 = new HashMap<String, AttributeValue>();
+    dynamoRecord2.put("!-\"%", null);
+    Assert.assertNull(DynamoUtils.safeGetString(dynamoRecord2, "!-\"%"));
+
+    final HashMap<String, AttributeValue> dynamoRecord3 = new HashMap<String, AttributeValue>();
+    Assert.assertNull(DynamoUtils.safeGetString(dynamoRecord3, ""));
+
+    final HashMap<String, AttributeValue> dynamoRecord4 = new HashMap<String, AttributeValue>();
+    AttributeValue attribute = DynamoUtils.createAttributeValue("test");
```
Comment (peterschrammel) [inline amazon-kinesis-client/src/test/java/software/amazon/kinesis/leases/DynamoUtilsTest.java:27]: final

File: amazon-kinesis-client/src/test/java/software/amazon/kinesis/leases/DynamoUtilsTest.java [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+
+public class DynamoUtilsTest {
+
+  @Test
+  public void testSafeGetString() {
+    final HashMap<String, AttributeValue> dynamoRecord = new HashMap<String, AttributeValue>();
```
Comment (peterschrammel) [inline amazon-kinesis-client/src/test/java/software/amazon/kinesis/leases/DynamoUtilsTest.java:15]: `String, AttributeValue` could be omitted on the rhs.

**Review by peterschrammel — APPROVED** [review]

**Pull request closed without merging**